Travis: error: Could not compile `sync15-adapter`.

We need to investigate that failure 

```
error[E0277]: the trait bound `failure::Error: std::error::Error` is not satisfied
  --> sync15-adapter/src/error.rs:67:17
   |
67 | #[derive(Debug, Fail)]
   |                 ^^^^ the trait `std::error::Error` is not implemented for `failure::Error`
   |
   = note: required because of the requirements on the impl of `failure::Fail` for `failure::Error`
   = note: required for the cast to the object type `failure::Fail`
error: aborting due to previous error
For more information about this error, try `rustc --explain E0277`.
error: Could not compile `sync15-adapter`.
Caused by:
```

--- a/sync15-adapter/Cargo.toml
+++ b/sync15-adapter/Cargo.toml
@@ -16,8 +16,8 @@ hyper = "0.11"
 log = "0.4"
 lazy_static = "1.0"
 base16 = "0.1"
-failure = "0.1.1"
-failure_derive = "0.1.1"
+failure = "= 0.1.1"
+failure_derive = "= 0.1.1"
 
 [dev-dependencies]
 env_logger = "0.5"

